content/k8s/fluxcd/helmrepository/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+content_type: project
+flavours:
+- none
+from_repo: k8s/manual-app-deployment/project-overview
+prerequisites:
+  hard:
+  - k8s/manual-app-deployment/project-overview
+  soft:
+  - k8s/fluxcd/kustomization
+ready: true
+submission_type: continue_repo
+tags:
+- kubernetes
+- fluxcd
+title: Helmrepository
+---
+
+## Helmrepository
```

Comment:
Can you capitalize the first letter of each paragraph, add ending periods and capitalize words like Flux?

Also, whenever mentioning a file withing a text paragraph, add the back quotes (``) so it gets highlighted, easier for the students to spot it.